### PR TITLE
doc fix of HiveThriftServer

### PR DESCRIPTION
**Pull request by zjffdu**
## What changes were proposed in this pull request?

Just minor doc fix. 

\cc @yhuai

### CODE REVIEW

**Comment by srowen** [comment]
LGTM, are those all the copy / paste errors?

**Comment by zjffdu** [comment]
I think so,  Just post another commit for another place's doc fix.

File: sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerSessionPage.scala [hunk 1]
```diff
@@ -29,7 +29,7 @@ import org.apache.spark.sql.hive.thriftserver.HiveThriftServer2.{ExecutionInfo,
 import org.apache.spark.ui._
 import org.apache.spark.ui.UIUtils._
 
-/** Page for Spark Web UI that shows statistics of a streaming job */
+/** Page for Spark Web UI that shows statistics of jobs running in thrift server */
```
Comment (srowen) [inline sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerSessionPage.scala]: Nit: in *the* thrift server.

**Comment by srowen** [comment]
Merged to master/2.0

**Pull request closed without merging**